### PR TITLE
Remove multiple cluster supports and simplify FC templates

### PR DESCRIPTION
**Pull request by WilliamMcCumstie**
`flight-inventory` will only be installed in single cluster mode. Therefore multiple cluster mode has been removed from the CLI. The cluster is still embedded in the file paths, however it is now always the `default` cluster.

Also simplified the FC templates so they are only `erb`/`markdown` (as opposed to `erb`/`yaml`/`markdown`)

### CODE REVIEW

File: lib/inventoryware/utils.rb [hunk 1]
```diff
@@ -59,12 +59,7 @@ def self.check_file_location?(path)
 
     # raise an error if given path isn't a directory
     def self.exit_unless_dir(path)
```
Comment (benarmston) [inline lib/inventoryware/utils.rb:61]: This name isn't true anymore </pedantry>.

File: lib/inventoryware/utils.rb [hunk 2]
```diff
@@ -59,12 +59,7 @@ def self.check_file_location?(path)
...
-      unless File.directory?(path)
-        raise FileSysError, <<-ERROR.chomp
-Directory #{File.expand_path(path)} not found.
-Please create it before continuing"
-        ERROR
-      end
+      Dir.mkdir_p(path) unless File.directory?(path)
```
Comment (benarmston) [inline lib/inventoryware/utils.rb:62]: We probably ought to raise an exception if `path` exists but is not a directory.

File: lib/inventoryware/utils.rb (hunk 1 above)
Comment (WilliamMcCumstie) [reply lib/inventoryware/utils.rb]: This whole section of code is a bit over kill. It's all be removed/replaced with a single `FileUtils.mkdir_p` via the `Config`

File: lib/inventoryware/utils.rb (hunk 2 above)
Comment (WilliamMcCumstie) [reply lib/inventoryware/utils.rb]: ditto

File: lib/inventoryware/utils.rb (hunk 1 above)
Comment (benarmston) [reply lib/inventoryware/utils.rb]: The new commit looks much better.  Thanks.

**Pull request merged**